decidim/decidim
# Add new print permission in intiatives

#### :tophat: What? Why?
This PR adds a new Print permission for the intitiatives, so that we can decouple editing and printing of initiatives in 2 different actions.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #11849

#### Testing
1. Make sure your INITIATIVES_PRINT_ENABLED is set on false. 
2. See that you cannot print the initiative
4. Apply patch and restart the server 
5. See that you cannot print it
6. Change INITIATIVES_PRINT_ENABLED to true 
7. See that you can print the initiative. 

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!
